templates/js-template-default/frameworks/runtime-src/proj.android-studio/app/build.gradle

```diff
@@ -19,8 +19,7 @@ android {
                     // skip the NDK Build step if PROP_NDK_MODE is none
                     targets 'cocos2djs'
                     arguments 'NDK_TOOLCHAIN_VERSION=4.9'
-                    arguments 'APP_PLATFORM=android-'+PROP_TARGET_SDK_VERSION
```

Comment:
APP_PLATFORM isn't needed?

Reply:
yes, it always equal mini sdk version, refer to below. discuss at https://github.com/cocos2d/cocos2d-x/pull/18646

> @newnon Sounds reasonable, but android SDK will set it on build automatic? how to prove that, or is there some official docs said that?

> sure https://developer.android.com/ndk/guides/application_mk.html
>APP_PLATFORM
This variable contains the minimum Android platform version you want to support. For example, a value of android-15 specifies that your library uses APIs that are not available below Android 4.0.3 (API level 15) and can't be used on devices running a lower platform version. For a complete list of platform names and corresponding Android system images, see Android NDK Native APIs.
>Instead of changing this flag directly, you should set the minSdkVersion property in the defaultConfig or productFlavors blocks of your module-level build.gradle file. This makes sure your library is used only by apps installed on devices running an adequate version of Android. The ndk-build toolchain uses the following logic to choose the minimum platform version for your library based the ABI you're building and the minSdkVersion you specify:
>If there exists a platform version for the ABI equal to minSdkVersion, ndk-build uses that version.
Otherwise, if there exists platform versions lower than minSdkVersion for the ABI, ndk-build uses the highest of those platform versions. This is a reasonable choice because a missing platform version typically means that there were no changes to the native platform APIs since the previous available version.
Otherwise, ndk-build uses the next available platform version higher than minSdkVersion.